NetSuiteProject/Exercise.js

```diff
@@ -2,22 +2,65 @@
 function add(x,y){
 	return x + y;
 };
+
+function addition(x,y){
+	if(typeof add (x,y) === "string"){
+	alert("This function only accepts Integers")
+	}
+	else {return x + y;}
+}
+
+
 //subtract
 function subtract(x, y){
 	return (x - y);
 };
+
+function subtraction(x, y){
+	if(isNaN(subtract(x,y))  === true){
```

Comment:
Here (and in the rest of the new functions) you're checking if the result of `subtract` is `NaN`, but in `addition`, it looks like you are trying to check if the result of `add` is a `String`. Why the difference?